linkerd/core/src/main/scala/io/buoyant/linkerd/AnnouncerInitializer.scala

```diff
@@ -0,0 +1,12 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.twitter.finagle.Announcer
+import io.buoyant.config.ConfigInitializer
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "kind")
+trait AnnouncerConfig {
+  def mk(): Announcer
+}
+
+trait AnnouncerInitializer extends ConfigInitializer
```

Comment:
In this example I have two servers that may route over two local services...  my point is there's not necessarily a 1:1 relationship between servers and services